README.md

```diff
@@ -46,48 +46,34 @@ This required re-writing the history of the repo. You **must** [pull and rebase
 If you want to contribute to the code, please fork the repository, make your changes, and create a pull request.
 
 ### Local setup
+Build the necessary services:
 
-Install the [PostgreSQL](https://www.postgresql.org/) database server. It's a useful idea to setup [passwordless authentication for local connections](https://www.postgresql.org/docs/current/static/auth-methods.html#AUTH-TRUST).
+    sudo docker-compose build
 
-You'll also need the psql and libxml development libraries. 
+Setup the database:
 
-```sh
-# Ubuntu/debian
-sudo apt-get install libpq-devel libxml2-dev libxslt1-dev python-dev
-# macOS
-brew install libxml2
-```
-
-You'll need python 2.7 and [virtualenv](https://virtualenv.pypa.io/en/stable/installation/).
-
-Clone this repo, and setup a virtualenv:
-
-    virtualenv --no-site-packages env
-    source env/bin/activate
-
-Install requirements:
-
-    pip install -r requirements.txt
+    sudo docker-compose run --rm web python setup_database.py
+    sudo docker-compose run --rm web python app.py db stamp head
 
-Add the following lines to your `.hosts` file:
+Start the server:  
 
-    127.0.0.1 api.pmg.test
```

Comment:
still need these

Reply:
I added it to compose file, so we don't need it here

Reply:
I still need this to be able to view the homepage - how else can I view the site?

![Screenshot_2019-08-12_14-08-55](https://user-images.githubusercontent.com/235801/62863985-becded00-bd0a-11e9-865c-0f719b1f56b5.png)

Reply:
I added extra_hosts to docker-compose.yml

Reply:
![image](https://user-images.githubusercontent.com/33839033/62864132-93033500-bd13-11e9-9ba7-5a2c9a622c19.png)

Reply:
![image](https://user-images.githubusercontent.com/33839033/62864247-d9589400-bd13-11e9-92c9-bc8379f25963.png)